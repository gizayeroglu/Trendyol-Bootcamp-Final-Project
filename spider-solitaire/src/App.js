import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import HomePage from './pages/HomePage/HomePage';
import GamePage from './pages/GamePage/GamePage';

function App() {
  return (
    <Router>
      <Switch>
        <Route path='/' exact component={HomePage} />
        <Route path='/play' exact component={GamePage} />
      </Switch>
    </Router>
  );
}

export default App;
