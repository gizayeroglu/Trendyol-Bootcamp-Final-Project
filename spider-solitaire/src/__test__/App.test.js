import React from 'react';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import { shallow, configure } from 'enzyme';

import App from '../App';

configure({adapter: new Adapter()});

describe('App test', function () {

  let wrapper;

  beforeEach(() => { 
    wrapper = shallow(<App />);
  })

  it('should render App', () => {
    expect(wrapper).toMatchSnapshot();
  })
})