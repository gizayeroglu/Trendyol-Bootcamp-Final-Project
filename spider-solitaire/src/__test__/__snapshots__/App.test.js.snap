// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test should render App 1`] = `ShallowWrapper {}`;
